fluentui_others/src/main/res/values-night/themes.xml

```diff
@@ -74,11 +57,7 @@
         <item name="fluentuiDialogBackgroundColor">?attr/fluentuiBackgroundSecondaryColor</item>
         <item name="fluentuiDialogCloseIconColor">?attr/colorPrimary</item>
 
-        <!--Drawer-->
-        <item name="fluentuiDrawerBackgroundColor">?attr/fluentuiBackgroundSecondaryColor</item>
-        <item name="fluentuiDrawerHandleColor">@color/fluentui_gray_500</item>
-
-        <!--ListItemView-->
+         <!--ListItemView-->
```

Comment:
nit:

Reply:
Corrected. ListItemView actually should be in listitem module, corrected this